.gitignore

```diff
@@ -6,3 +6,4 @@
 /dev/
 /docs/build/
 /docs/site/
+/docs/Manifest.toml
```

Comment:
One needs to version control the `/docs/Manifest.toml`
so that you can `] dev ..` into it, so that the dev version of the docs is correct